src/utils/components/NetworkInterfaceModal/components/NetworkInterfaceNetworkSelect/NetworkInterfaceNetworkSelect.tsx

```diff
@@ -54,6 +54,7 @@ const NetworkInterfaceNetworkSelect: FC<NetworkInterfaceNetworkSelectProps> = ({
   const { t } = useKubevirtTranslation();
   const vmiNamespace = vm?.metadata?.namespace || namespace;
   const { loaded, loadError, nads } = useNADsData(vmiNamespace);
+  const [selectedFirstOnLoad, setSelectedFirstOnLoad] = useState(false);
```

Comment:
Can this be a ref? So we'll not trigger a re-rendering? and would not go into the useEffect dependencies

Reply:
IMHO we need a re-render to make sure  we re-create the network select component - but I can check if `handleChange` guarantees that.